### PR TITLE
Fix Errors in Swift 2 / Xcode 7

### PR DESCRIPTION
**Pull request by iOSCowboy**
The pod is unable to compile in Xcode 7. 
This commit fixes the issues introduced in Xcode 7 / Swift 2.0.

### CODE REVIEW

**Comment by iOSCowboy** [comment]
pig @mikaoj

File: Pod/Classes/Controller/PhotosViewController.swift [hunk 1]
```diff
@@ -407,7 +407,7 @@ final class PhotosViewController : UICollectionViewController, UIPopoverPresenta
     
     private func updateAlbumTitle(album: PHAssetCollection) {
         // Update album title
-        albumTitleView?.albumTitle = album.localizedTitle
+        albumTitleView?.albumTitle = album.localizedTitle!
```
Comment (mikaoj) [inline Pod/Classes/Controller/PhotosViewController.swift]: I'm not a fan of forced unwrapping

File: Pod/Classes/Controller/ZoomAnimator.swift [hunk 2]
```diff
@@ -45,10 +45,10 @@ final class ZoomAnimator : NSObject, UIViewControllerAnimatedTransitioning {
             destinationImageView.hidden = true
             toViewController.view.alpha = 0.0
             fromViewController.view.alpha = 1.0
-            containerView.backgroundColor = toViewController.view.backgroundColor
+            containerView!.backgroundColor = toViewController.view.backgroundColor
```
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: Forced unwrapping

File: Pod/Classes/Controller/ZoomAnimator.swift [hunk 3]
```diff
@@ -45,10 +45,10 @@ final class ZoomAnimator : NSObject, UIViewControllerAnimatedTransitioning {
...
             
             // Setup scaling image
-            let scalingFrame = containerView.convertRect(sourceImageView.frame, fromView: sourceImageView.superview)
+            let scalingFrame = containerView!.convertRect(sourceImageView.frame, fromView: sourceImageView.superview)
```
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: Forced unwrapping

File: Pod/Classes/Controller/ZoomAnimator.swift [hunk 4]
```diff
@@ -62,8 +62,8 @@ final class ZoomAnimator : NSObject, UIViewControllerAnimatedTransitioning {
             }
             
             // Add views to container view
-            containerView.addSubview(toViewController.view)
-            containerView.addSubview(scalingImage)
+            containerView!.addSubview(toViewController.view)
```
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: Forced unwrapping ;)

File: Pod/Classes/Controller/ZoomAnimator.swift [hunk 5]
```diff
@@ -62,8 +62,8 @@ final class ZoomAnimator : NSObject, UIViewControllerAnimatedTransitioning {
...
+            containerView!.addSubview(scalingImage)
```
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: .

File: Pod/Classes/Factory/AlbumCellFactory.swift [hunk 6]
```diff
@@ -31,8 +31,6 @@ final class AlbumCellFactory : TableViewCellFactory {
     
     func cellForIndexPath(indexPath: NSIndexPath, withDataSource dataSource: SelectableDataSource, inTableView tableView: UITableView) -> UITableViewCell {
         let cell = tableView.dequeueReusableCellWithIdentifier(albumCellIdentifier, forIndexPath: indexPath) as! AlbumCell
-        let cachingManager = PHCachingImageManager.defaultManager() as! PHCachingImageManager
-        cachingManager.allowsCachingHighQualityImages = false
```
Comment (mikaoj) [inline Pod/Classes/Factory/AlbumCellFactory.swift]: Seems like some functionality got lost here

File: Pod/Classes/Model/FetchResultsDataSource.swift [hunk 7]
```diff
@@ -127,9 +127,9 @@ final class FetchResultsDataSource : NSObject, SelectableDataSource, PHPhotoLibr
                 
                 if incrementalChange {
                     // Incremental change, tell delegate what has been deleted, inserted and changed
-                    removedIndexPaths = indexPathsFromIndexSet(collectionChanges.removedIndexes, inSection: index)
-                    insertedIndexPaths = indexPathsFromIndexSet(collectionChanges.insertedIndexes, inSection: index)
-                    changedIndexPaths = indexPathsFromIndexSet(collectionChanges.changedIndexes, inSection: index)
+                    removedIndexPaths = indexPathsFromIndexSet(collectionChanges.removedIndexes!, inSection: index)
```
Comment (mikaoj) [inline Pod/Classes/Model/FetchResultsDataSource.swift]: Forced unwrapping

File: Pod/Classes/View/SelectionView.swift [hunk 8]
```diff
@@ -63,11 +63,12 @@ import UIKit
         
         //// Bezier Drawing (Picture Number)
         CGContextSetFillColorWithColor(context, UIColor.whiteColor().CGColor)
-        let size = selectionString.sizeWithAttributes(settings.selectionTextAttributes)
+        
+        let size = selectionString.sizeWithAttributes(settings.selectionTextAttributes as? [String: AnyObject])
```
Comment (mikaoj) [inline Pod/Classes/View/SelectionView.swift]: Change the type in settings instead of casting

**Comment by mikaoj** [comment]
Great! Made some comments, let me know if you want to fix them...or I'll just do it myself :)

File: Pod/Classes/Controller/PhotosViewController.swift (hunk 1 above)
Comment (iOSCowboy) [inline Pod/Classes/Controller/PhotosViewController.swift]: What is the alternative? `if`-ing the existence of the value first?

**Comment by iOSCowboy** [comment]
I've fixed the straight-forward ones, @mikaoj. Would you mind having a look at the unwrapping?

Thanks for the library!

File: Pod/Classes/Controller/PhotosViewController.swift (hunk 1 above)
Comment (mikaoj) [inline Pod/Classes/Controller/PhotosViewController.swift]: if let title = localizedTitle ...

File: Pod/Classes/Controller/ZoomAnimator.swift (hunk 2 above)
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: Use ? instead

File: Pod/Classes/Controller/ZoomAnimator.swift (hunk 4 above)
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: And also one line went missing :)

File: Pod/Classes/Controller/ZoomAnimator.swift (hunk 5 above)
Comment (mikaoj) [inline Pod/Classes/Controller/ZoomAnimator.swift]: ?

**Pull request merged**